tests/factories/cht/contacts/person.js

```diff
@@ -0,0 +1,39 @@
+const Factory = require('rosie').Factory;
+const uuid = require('uuid');
+
+const parent = {
+  _id: 'clinic1',
+  parent: {
+    _id: 'hc1',
+    parent: {
+      _id: 'dist1'
+    }
+  }
+};
```

Comment:
Instead of making a sparse object here, please use the clinic factory. For an example of how to do this, take a look at the [docs](https://github.com/rosiejs/rosie), specifically the first code sample and the part about "Default to two players. If players were given, fill in"

The idea with these factories is that for any case where you find a need to make up some object by hand, instead use the factory to fill it in, and that includes in the factories themselves. I'm putting this comment here on this one, but please apply to others with the same idea.